aptos-node/src/lib.rs

```diff
@@ -61,6 +61,10 @@ pub struct AptosNodeArgs {
     #[clap(long, parse(from_os_str), requires("test"))]
     test_dir: Option<PathBuf>,
 
+    /// Path to node configuration file override for local test mode. Cannot be used with --config
+    #[clap(long, parse(from_os_str), requires("test"), conflicts_with("config"))]
+    config_override: Option<PathBuf>,
+
```

Comment:
I don't think you necessarily need this, it'll override all fields with default fields if you don't them already